core/trino-parser/pom.xml

```diff
@@ -13,7 +13,6 @@
 
     <properties>
         <air.main.basedir>${project.parent.basedir}</air.main.basedir>
-        <project.build.targetJdk>8</project.build.targetJdk>
```

Comment:
Could we do it as a separate commit, kind of like a cherry-on-top? :)

Reply:
What’s the benefit? It’s part of the refactoring — the parts that depend on Java 8 moved to the grammar module, so that constraint moved there, too, at the same time.